packages/grafana-plugin-configs/utils.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import { glob } from 'glob';
+import path from 'path';
+import process from 'process';
+
+export function getPackageJson() {
+  return require(path.resolve(process.cwd(), 'package.json'));
+}
+
+export function getPluginJson() {
+  return require(path.resolve(process.cwd(), 'plugin.json'));
+}
+
+// Support bundling nested plugins by finding all plugin.json files in src directory
+// then checking for a sibling module.[jt]sx? file.
```

Comment:
nit: I think we can remove these comments as the getEntries function will not work with nested plugins now.